### ISSUE TITLE
ADMIN_ENABLE variable not used

### ISSUE THREAD

**Issue by eduramiba** [getsentry/onpremise#1039]
### Version

21.7.0

### Steps to Reproduce

Setting variable ADMIN_ENABLE = False does not disable the Django admin panel.

`Django.contrib.admin` is unconditionally added here (https://github.com/getsentry/sentry/blob/a741ce3cadef2537932e9d1262a723b9559bd738/src/sentry/conf/server.py#L322) but should not be.

Thank you,
Eduardo

### Expected Result

The admin panel should not be available.

### Actual Result

The admin panel is available.

**Comment by chadwhitacre** [comment]
[Seems legit](https://stackoverflow.com/questions/4845239/how-can-i-disable-djangos-admin-in-a-deployed-project-but-keep-it-for-local-de), but on the other hand I reeeeeeally don't think we have the admin panel in SaaS. How do we prevent it from showing up there? Does that give us a clue for how to disable in self-hosted?

**Comment by eduramiba** [comment]
> [Seems legit](https://stackoverflow.com/questions/4845239/how-can-i-disable-djangos-admin-in-a-deployed-project-but-keep-it-for-local-de), but on the other hand I reeeeeeally don't think we have the admin panel in SaaS. How do we prevent it from showing up there? Does that give us a clue for how to disable in self-hosted?

On self-hosted I had to manually edit the server.py file inside the `web` docker image and it worked.

**Comment by chadwhitacre** [comment]
Yeah, I think for SaaS we're turning off the admin in our [private `getsentry` wrapper](https://develop.sentry.dev/sentry-vs-getsentry/).

For self-hosted, `urls.py` in `sentry` doesn't give us anything, it [goes off `INSTALLED_APPS`](https://github.com/getsentry/sentry/blob/b3ced8f3fba5cbb5c47d08516ce43ab8ce014313/src/sentry/conf/urls.py#L18-L21).

Seems like a valid ask, @eduramiba, thanks for the ticket.
